GNUmakefile

```diff
@@ -244,3 +250,466 @@ deploy-images:
 	@IMAGE=$(ENGINE_IMG) ./build/push
 	@IMAGE=$(PAIR_IMG) ./build/push
 	@IMAGE=$(DS_IMG) ./build/push
+	@echo ""
+	@echo "Pushing developer image...."
```

Comment:
Let's add following targets to `.travis.yml`:
- make dev-image
- make lint
- make format

Reply:
Can you merge existing `golint` to `lint`
https://github.com/kubemove/kubemove/blob/4a3cf00edc9853dcf40e23836a8b47506bee945d/GNUmakefile#L50

Reply:
done